<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyFeaturesForm</class>
 <widget class="QWidget" name="KeyFeaturesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Tmon몬소리 Black</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {color: rgb(180, 199, 231)};
</string>
           </property>
           <property name="text">
            <string>Key Features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="savePushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton {background-color: rgb(170, 255, 255)};

</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="zoomInCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}
</string>
            </property>
            <property name="text">
             <string>Zoom In</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="zoomOutCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Zoom Out</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="leftRotateCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>90° Rotate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rightRotateCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>-90° Rotate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="brushCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Brush</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rectangleCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Rectangle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="triangleCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Triangle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ellipseCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Ellipse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="moveShapeCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Move Shape</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="blendingCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Blending</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="lengthMeasurementCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Length Measurement</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="angleMeasurementCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Angle Measurement</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="brightnessCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Brightness</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="darknessCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Darkness</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sharpenCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Sharpen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="contrastCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Contrast</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="inversionCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Inversion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="horizontalFlipCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Horizontal Flip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="verticalFlipCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Vertical Flip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="implantCheckBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox {color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string>Implant</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
